Sources/GreenstandWalletSDK/GreenstandWalletSDK.swift

```diff
@@ -107,4 +110,19 @@ public class GreenstandWalletSDK: GreenstandWalletSDKProtocol {
 
         completion(.success(TokenTransferReceipt()))
     }
+    public func getTokensIn(walletNamed name: String, completion: @escaping (Result<[Token], Error>) -> Void) {
```

Comment:
Guess we dont need the wallet name here as we are using the authenticatedWalletName value